frontend/src/components/config/Route/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+
+export function CustomRoute(props: any) {
+  const { component: Component, layout: Layout, ...rest } = props;
+  return (
+    <Route
+      {...rest}
+      render={(renderProps) => (
```

Comment:
I'll probably keep it just because it's an ESLint rule (`react/jsx-max-props-per-line`) but I don't feel too strongly about it either so we can always disable it in the future if it gets too annoying.